### PR TITLE
6124 get data actor screen

### PR DESCRIPTION
**Pull request by dinhquang251**
GetData for Actor Screen
https://edu-redmine.framgia.vn/issues/6124

### CODE REVIEW

**Comment by dinhquang251** [comment]
ready

File: app/src/main/res/layout/item_genre.xml [hunk 1]
```diff
@@ -18,14 +18,13 @@
...
         <ImageView
-            bindImageGenre="@{viewModel.mGenresObservableField."
             android:layout_width="@dimen/dp_400"
             android:layout_height="@dimen/dp_200"
             android:scaleType="fitXY"
             android:src="@color/color_red_dark" />
 
         <TextView
-            android:layout_width="wrap_content"
+            android:layout_width="10dp"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/item_genre.xml]: hardcode

File: app/src/main/res/layout/fragment_actor_info.xml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/title_over_view"
+                android:textSize="@dimen/dp_20"
+                android:textStyle="bold"
+                app:layout_constraintLeft_toLeftOf="@id/text_birthday"
+                app:layout_constraintTop_toBottomOf="@id/text_birthday" />
+
+            <TextView
+                android:layout_width="match_parent"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/fragment_actor_info.xml]: not match parent

File: app/src/main/res/layout/fragment_actor_info.xml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                android:textSize="@dimen/sp_30"
+                android:textStyle="bold"
+                app:layout_constraintLeft_toLeftOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <TextView
+                android:id="@+id/text_actor_name"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="Name: "
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/fragment_actor_info.xml]: hardcode

File: app/src/main/res/layout/fragment_actor_info.xml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+                android:text="@{viewModel.personObservable.get().getName()}"
+                android:textSize="@dimen/dp_15"
+                app:layout_constraintLeft_toRightOf="@id/text_actor_name"
+                app:layout_constraintTop_toTopOf="@id/text_actor_name" />
+
+            <TextView
+                android:id="@+id/text_birthday"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="Birthday: "
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/fragment_actor_info.xml]: hardcode

**Comment by dinhquang251** [comment]
ready

**Pull request merged**